airbyte-integrations/connectors/source-chargebee/source_chargebee/manifest.yaml

```diff
@@ -312,9 +312,14 @@ definitions:
     transformations:
       - type: CustomTransformation
         class_name: source_chargebee.components.CustomFieldTransformation
+      - type: AddFields
+        fields:
+          - path: ["subscription_id"]
+            value_type: string
+            value: "{{ stream_slice.subscription_id }}"
     $parameters:
       name: "subscription_with_scheduled_changes"
-      primary_key: "id"
+      primary_key: "subscription_id"
```

Comment:
This is normally a breaking change but since it wasn't working before (`id` wasn't populated), then I won't make it a breaking change

Reply:
Should we contact Chargebee support to see if the endpoint should/will start returning the subscription ID? Is it possible that our sandbox is an anomaly and some accounts *do* already return the subscription ID?

Reply:
The subscription ID is already [there](https://github.com/airbytehq/airbyte/blob/696b9795aaad5ef84a5f885b0ca7257ae66b3344/airbyte-integrations/connectors/source-chargebee/unit_tests/resource/http/response/subscription_with_scheduled_changes.json#L38). We could have done the transformation differently but I think it needs to be on the root level